opm/simulators/flow/equil/EquilibrationHelpers.hpp

```diff
@@ -453,9 +453,6 @@ class RsSatAtContact : public RsFunction<typename FluidSystem::Scalar>
     /**
      * Function call.
      *
-     * \param[in] depth Depth at which to calculate RS
-     * value.
-     *
```

Comment:
I think it might be better to include the parameter name in the function declaration instead of removing the documentation. While it is true that the depth parameter is unused in the function body, it is still a part of the interface.

Reply:
ok.